Directory.Build.props

```diff
@@ -23,7 +23,7 @@
  
     
     <UwpMetaPackageVersion>5.4.1</UwpMetaPackageVersion>
-    <DefaultTargetPlatformVersion>16299</DefaultTargetPlatformVersion>
+    <DefaultTargetPlatformVersion>17125</DefaultTargetPlatformVersion>
```

Comment:
We should probably wait until the final SDK release and do this once, eh?

Reply:
Yeah, I had to change it because the app would crash when trying to use the native control otherwise. Maybe we could remove the code to enable the feature and leave a PR with it to merge when the final SDK is released, along with this update. What do you think?

Reply:
@nmetulev thoughts?